dev/core/src/com/google/gwt/dev/jjs/impl/gflow/cfg/CfgBuilder.java

```diff
@@ -645,6 +627,16 @@ public boolean visit(JStatement x, Context ctx) {
       throw new UnsupportedNodeException(x.getClass().toString());
     }
 
+    @Override
+    public boolean visit(JSwitchExpression x, Context ctx) {
```

Comment:
Wondering about the consequences given that JSwitchStatement does quite a bit.

Reply:
Oh I see. I just found https://github.com/gwtproject/gwt/commit/ea7228a75491f76b1240a7648eb48d198b19acc4

So maybe we should officially deprecate it for removal (or fix its bugs and keep it).